app/src/main/java/com/bridgewell/quickstart/android/activity/TabbedViewActivity.kt

```diff
@@ -0,0 +1,191 @@
+                    tab.setIcon(R.drawable.ic_files) // Set icon for Files tab
+                }
+                3 -> {
+                    tab.text = "User"
+                    tab.setIcon(R.drawable.ic_user) // Set icon for User tab
+                }
+            }
+        }.attach()
+
+        // Set default selected tab
+        viewPager.setCurrentItem(0, false)
+
+        // Get the selected tab index from the intent
+        val selectedTabIndex = intent.getIntExtra("SELECTED_AD_TYPE", 0)
+
+        // Parse the index back to the AdType enum
+        val selectedAdType = AdType.entries[selectedTabIndex]
+
+        for (i in 1..17) {
+            val imageViewId = resources.getIdentifier("imageView$i", "id", packageName)
+            val imageView = findViewById<ImageView>(imageViewId)
+            val skeletonColor = when (i) {
+                1, 2, 5, 6, 11, 10 -> R.color.skeleton_gray_1
+                else -> R.color.skeleton_gray_2
+            }
+            imageView.loadSkeleton {
+                color(skeletonColor)
+                cornerRadius(4F)
+            }
+        }
+
+        createAd(selectedAdType)
+    }
+
+    private fun createAd(adType: AdType) {
+        if (adType == AdType.BANNER) {
+            val imageViewBanner = findViewById<ImageView>(R.id.imageView1)
+            imageViewBanner.hideSkeleton()
+        }
+
+        val listener = object : BwsAdViewListener { // Common listener
+            override fun onAdViewStartLoad(bannerView: BwsAdView?) {
+                showToast("onAdStartLoad")
+            }
+
+            override fun onAdViewLoaded(bannerView: BwsAdView?) {
+                showToast("onAdLoaded")
+            }
+
+            override fun onAdViewDisplayed(bannerView: BwsAdView?) {
+                showToast("onAdDisplayed")
+            }
+
+            override fun onAdViewFailed(bannerView: BwsAdView?, exception: AdException?) {
+                showToast("onAdFailed ${exception?.message}")
+                Timber.d("onAdFailed ${exception?.message}")
+            }
+
+            override fun onAdViewClicked(bannerView: BwsAdView?) {
+                showToast("onAdClicked")
+            }
+
+            override fun onAdModalBrowserClosed(bannerView: BwsAdView?) {
+                showToast("onAdModalBrowserClosed")
+            }
+
+            override fun onAdViewClosed() {
+                showToast("onAdViewClosed")
+            }
+        }
+
+        when (adType) {
+            AdType.POP_UP_AD -> {
+                inAppApi.createBwsPopupAd(
+                    this,
+                    configID = CONFIG_ID_BANNER,
+                    refreshTimeSeconds = 0,
+                    listener = listener
+                )
+            }
+            AdType.STICKY_BOTTOM_RIGHT -> {
+                inAppApi.createBwsRightSideStickyAd(
+                    this,
+                    configID = CONFIG_ID_BANNER,
+                    bottomMargin = 77,
```

Comment:
Cannot find a parameter with this name: bottomMargin

Reply:
in the new Figma design, the bottom margin changed. Add an optional parameter to the BWSDK to be able to adjust the bottom margin of bottom-right sticky banner. Please check my new PR